cloud/aws/deploy/api/apigateway.go

```diff
@@ -221,3 +242,13 @@ func awsOperation(op *openapi3.Operation, funcs map[string]string) *openapi3.Ope
 
 	return op
 }
+
+func toPulumiStrArray(strs []string) pulumi.StringArray {
+	arr := pulumi.StringArray{}
+
+	for _, v := range strs {
+		arr = append(arr, pulumi.String(v))
+	}
+
+	return arr
+}
```

Comment:
doesn't `pulumi.ToStringArray()` do this?

https://pkg.go.dev/github.com/pulumi/pulumi/sdk/v3@v3.65.1/go/pulumi#ToStringArray

Reply:
Fixed